### PR TITLE
Issue #SB-21655 feat: Documenting desktop registry and update API

### PR DESCRIPTION
**Pull request by souravdey091**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

File: apis/desktop/app-update/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+---
+<!DOCTYPE html>
+<html>
+	<head>
+		{% include head.html %}
+		<meta name="robots" content="index, follow">
+		<script src="https://rebilly.github.io/ReDoc/releases/v1.19.1/redoc.min.js"> </script>
```
Comment (SMYALTAMASH) [inline apis/desktop/app-update/index.html:8]: please update this JS URL with "https://sunbirdpublic.blob.core.windows.net/documentationassets/redoc.standalone.js"

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @souravdey091,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://616.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/app-update// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis/desktop/device-registry// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://616.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

**Pull request merged**